net-libs/libtorrent/libtorrent-0.13.6.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit libtool toolchain-funcs
+
+DESCRIPTION="BitTorrent library written in C++ for *nix"
+HOMEPAGE="https://rakshasa.github.io/rtorrent/"
+SRC_URI="http://rtorrent.net/downloads/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
```

Comment:
The reason this hasn't been done in the past is because the only program which uses this is `net-p2p/rtorrent` and each version of rtorrent is only usable with exactly one version of libtorrent that was released together with it. Because of that, even if the soname didn't change, the resulting program would be buggy if the rtorrent and libtorrent versions didn't match up correctly.

I'm fairly confident if we add a subslot now, in the future someone will see `net-p2p/rtorrent` using a version instead of a subslot, correct it and start introducing subtle bugs.